examples/example-cluster.yaml

```diff
@@ -26,10 +26,21 @@ spec:
 
   ## Specify additional pod specification
   # podSpec:
+  #   affinity:
+  #     podAntiAffinity:
+  #       preferredDuringSchedulingIgnoredDuringExecution:
+  #         weight: 100
+  #         podAffinityTerm:
+  #           topologyKey: "kubernetes.io/hostname"
+  #           labelSelector:
+  #             matchlabels: <cluster-labels>
   #   resources:
   #     requests:
   #       memory: 1G
   #       cpu:    200m
+  #   tolerations: []
+  #   priorityClassName:
+  #   serviceAccountName: default
```

Comment:
It's a good idea, but who creates that service account? I don't think it's good to give these permissions to the operator.

I think the creator of the MySQLCluster should provide a service account.

Reply:
I think you misunderstood me. By creating something else than `default` we are making sure we are not giving extra permission in the first place. Because generally folks have default as over loaded with so many permissions in this case we will have a new service account that has no permissions.

Later on we can use the same to give permissions via PSP because right now mysql pod request to run as root. Hence in a cluster with PSP installed this will be useful by default. So rather than giving permissions to the default service account which is a bad idea, here we will give permissions to the service account which is dedicated for this mysql workloads.

Reply:
I get that, but my concern is about who creates that dedicated service account? The operator acts in multiple namespaces (cross-namespace), so the service account can't be created when the operator is deployed (by the helm chart). 

By giving permissions to the operator service account to create service accounts, the operator service account will have too many permissions, also I'm not sure that this should be the operator job. This is my concern. Maybe I'm wrong here.

Reply:
Oh I thought that operator only worked in single namespace.

Will come up with a better strategy with the newly gained knowledge :-)